src/components/SearchForm/SearchForm.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import SearchForm from "./SearchForm";
+
+
+describe("SearchForm Component", () => {
+  const SearchFormMock = (initialSearchQuery: string, onSearch: jest.Mock) => {
```

Comment:
It is not a React component, so no need to capitalize it. Such functions are usually called `setup` or `mount` or something like this.